build.sbt

```diff
@@ -112,7 +112,7 @@ libraryDependencies ++= Seq(
 dependencyOverrides += "com.google.guava" % "guava" % "16.0.1"
 dependencyOverrides += "commons-codec" % "commons-codec" % "1.6"
 dependencyOverrides += "commons-logging" % "commons-logging" % "1.2"
-dependencyOverrides += "io.netty" % "netty-all" % "4.0.42.Final"
+dependencyOverrides += "io.netty" % "netty-all" % "4.1.17.Final"
 dependencyOverrides += "org.apache.httpcomponents" % "httpclient" % "4.5.2"
```

Comment:
Great. Could you update the others together consistently? Eg.
- httpclient -> 4.5.4
- httpcore -> 4.4.8